databricks_cli/secrets/api.py

```diff
@@ -28,8 +28,10 @@ class SecretApi(object):
     def __init__(self, api_client):
         self.client = SecretService(api_client)
 
-    def create_scope(self, scope, initial_manage_principal):
-        return self.client.create_scope(scope, initial_manage_principal)
+    def create_scope(self, scope, initial_manage_principal, scope_backend_type,
```

Comment:
In the future, please try to avoid making breaking API changes. These new required positional arguments break anything that use this API.
Prefer to add new arguments with default values that retain the previous functionality, or instead add new methods if that cannot be done.